package.json

```diff
@@ -873,7 +873,7 @@
     "expand-home-dir": "0.0.3",
     "find-java-home": "1.2.2",
     "follow-redirects": "^1.15.6",
-    "globby": "^13.2.2",
```

Comment:
Some of these dependencies are causing Mend alerts. Hoping the update will reduce the number.